docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,152 @@ sidebar_position: 2
+backrest-backup-hippo-9gtqf                   1/1     Running   0          13s
+hippo-597dd64d66-4ztww                        1/1     Running   0          3m33s
+hippo-backrest-shared-repo-66ddc6cf77-sjgqp   1/1     Running   0          4m27s
+```
+
+The database will be empty by default, it requires the schema and sample data to
+work with the application.  Now you can initialize the database with the schema
+and sample data using this command:
+
+```bash
+bash <(curl -s https://gist.github.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/04eb5fe3d7f393af5a6760b03d9a1a3f5c725077/init-database.sh)>
+```
+
+Now the database is ready to connect from application.  The next section
+explains how to configure application:
+
+## Application Deployment
+
+Now you can deploy the `spring-petclinic-rest` app with this `Deployment`
+configuration:
+
+```bash
+kubectl apply -f https://gist.github.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/04eb5fe3d7f393af5a6760b03d9a1a3f5c725077/app-deployment.yaml
+```
+
+You can port-forward the application port and try to access it from your local system
+
+```
+kubectl port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n petclinic-demo
+```
+
+You can open [http://localhost:9966/petclinic](http://localhost:9966/petclinic)
+
+You should see a [Swagger UI][swagger] where you can play with the API.
+
+Since the binding is not present in the application, you will see be any values in results.
+In the next section, you will see how to fix it.
+
+## Service Binding
+
+The application was not working as the bindings were not present in the app.
+
+Now you can create the ServiceBinding custom resource to inject the bindings:
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+    name: spring-petclinic-rest
+    namespace: petclinic-demo
+spec:
+    services:
+    - group: "crunchydata.com"
+      version: v1
+      kind: Pgcluster
+      name: hippo
+    - group: ""
+      version: v1
+      kind: Secret
+      name: hippo-hippo-secret
+    application:
+      name: spring-petclinic-rest
+      group: apps
+      version: v1
+      resource: deployments
+    mappings:
+    - name: type
+      value: "postgresql"
+```
+
+For the convenience, the above resource can be installed like this:
+
+```bash
+kubectl apply -f https://gist.github.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/04eb5fe3d7f393af5a6760b03d9a1a3f5c725077/service-binding.yaml
+```
+
+The [next section](../creating-service-bindings/creating-service-binding)
+explains the ServiceBinding configuration.
+
+You can port-forward the application port and access it from your local system
+
+```
+kubectl port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n petclinic-demo
+```
+
+You can open [http://localhost:9966/petclinic](http://localhost:9966/petclinic)
+
+You should see a [Swagger UI][swagger] where you can play with the API.
+
+## Summary
```

Comment:
Rewrite for better clarity and in simple present tense. Will review again after the rewrite.

Reply:
Done. 02e68639019341ab5970e1f33e583a470613443e